scripts/init

```diff
@@ -165,7 +146,6 @@ printenv >> /etc/environment
 configure_services
 #taken from https://unix.stackexchange.com/a/71511
 { register_node |& tee /dev/fd/3 | grep -q 'Label "group" is required for this registration type' && exit; } 3>&1
```

Comment:
We grep here for particular message in the logs, `wcli` use different message for that, we have to check which exactly message we use.